grails-app/domain/org/pih/warehouse/invoice/InvoiceItem.groovy

```diff
@@ -173,6 +173,9 @@ class InvoiceItem implements Serializable {
                 unitPrice: unitPrice,
                 totalAmount: totalAmount,
                 totalPrepaymentAmount: totalPrepaymentAmount,
+                orderAdjustment: orderAdjustment,
+                productName: product?.name,
+                translatedProductName: product?.translatedName
```

Comment:
I'm debating how to handle this so I think we should discuss this on a tech huddle next week.

Reply:
Options: 
1. As-is
1. Return a representation of the product that includes the translated name(s) by type  
    1. Product.toJson() - This could get expensive if we have to render this 100s of times
    2. Product.toAnotherJson() that is used less often 
1. Create a new ProductName DTO that takes the product and synonyms and renders an object in a particular locale that conforms to our naming rules (i.e. one display_name per locale). 
    2. productName: [displayName: "english display name", brandNames: ["english brand name 1", "english brand name 2"]   
1. There might be other options as well